services/web-server/src/data/GithubAccessToken.js

```diff
@@ -0,0 +1,38 @@
+const assert = require('assert');
+const Entity = require('azure-entities');
+
+const GithubAccessToken = Entity.configure({
+  version: 1,
+  partitionKey: Entity.keys.ConstantKey('githubAccessTokens'),
+  rowKey: Entity.keys.StringKey('userId'),
+  signEntities: true,
```

Comment:
I suspect encryption is even more important, since these are user credentials..

Reply:
The access token is encrypted [here](https://github.com/taskcluster/taskcluster/pull/1365/files/f8038b5297b243ef3d300639266709b6817983ec#diff-e87117fe4e535da5efe2d8d264b0be73R13). Did you have something else in mind?

Reply:
I didn't, I was being dumb, sorry!